encoding/preparation.md

```diff
@@ -162,8 +173,23 @@ VapourSynth-Script.
     extension for Jupyter Notebook.
 
     ![A Jupyter Notebook.](images/cnvimage101.png)
-
-3.  [AvsPmod](https://github.com/AvsPmod/AvsPmod/releases). This is the
+    
+3.  [VapourSynth Multi-Viewer](https://github.com/mysteryx93/VapourSynthViewer.NET/releases/).
+     Multi-Viewer is a very simple and elementary previewing tool. 
+     While the text editing is vastly inferior to VSEdit’s, 
+     and the preview lacks a lot of VSEdit’s advanced features,
+     its tab-based previewing functionality easily outclasses VSEdit’s single preview window, 
+     because it makes comparisons between different script versions 
+     a lot more convenient and efficient.
+     
+     In short: very useful for thorough comparisons and filter fine-tuning, 
+     not so much for everything else.
+     
+     ![VS Multi-Viewer’s editing window.](images/VS-Multi-Viewer_Preview.png)
+     ![VS Multi-Viewer’s preview window.](images/VS-Multi-Viewer_Editor.png)
```

Comment:
Just add a newline before the second image and we should be good.

Reply:
This is kind of interesting, by having those lines indented 5 spaces instead of the 4 with the rest of the list items, they don't display a caption and also break the rendered numbering. AvsPmod here becomes item `1.` again, even though it's 4 in the md.